README.md

```diff
@@ -147,6 +147,8 @@ nslookup ${He_Name}.azurecr.io
   - One for App Service or AKS, Key Vault and Azure Monitor
   - One for Cosmos DB
 
+Some environment variables will save commands as strings instead of running the command and saving the output. This is intentional to avoid saving sensitive data in environment variables. For example, `export Imdb_RW_Key='az cosmosdb keys list ...'`. Make sure to run the export commands as is.
```

Comment:
add a sentence on how to use eval to get the actual value

Reply:
should we highlight this with >  ?  it's important and it kind of gets lost imo same comment for IMDb